.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: enabled
+dist: trusty
+
+language: node_js
+node_js:
+  - "node"
+install:
+  - npm install
+  - npm test
+notifications:
+  slack: artris:fjIt73ICT1EVM4sZNwbBElXx
```

Comment:
why slack?

Reply:
Status on builds will be posted to the lqsy-jar slack channel